exportify.js

```diff
@@ -216,8 +221,30 @@ let PlaylistExporter = {
 			})
 		})
 
+
+		// Step 3: Fetch album details - another wave of traffic, 20 albums at a time max
+		let album_promise = data_promise.then(() => {
+			album_ids = Array.from(album_ids)
+			let album_chunks = []; while (album_ids.length) { album_chunks.push(album_ids.splice(0, 20)) }
+			let album_promises = album_chunks.map((chunk_ids, i) => utils.apiCall(
+				'https://api.spotify.com/v1/albums?ids=' + chunk_ids.join(','), access_token, 100 * i))
+			return Promise.all(album_promises).then(responses => {
+				let album_details = {}
+				responses.forEach(response => response.albums.forEach(album => {
+					if (album) {
+						album_details[album.id] = {
+							label: album.label,
+							total_tracks: album.total_tracks,
+							album_type: album.album_type
+						};
+					}
+				}));
+				return album_details
+			});
+		});
```

Comment:
I got rid of most of the semicolons in this file, since they're apparently optional in most places in JavaScript. I'll probably come back and modify the style of this, but it seems right.